.github/workflows/deploy-vercel-staging.yml

```diff
@@ -254,7 +230,7 @@ jobs:
         env:
           VERCEL_TOKEN: ${{ secrets.VERCEl_TOKEN }}
         with:
-          deployment-url: ${{ env.VERCEL_DEPLOYMENT }} # Must only contain the domain name (no http prefix, etc.)
+          deployment-url: ${{ fromJson(steps.restore-variable.outputs.variables).VERCEL_DEPLOYMENT_DOMAIN }} # Must only contain the domain name (no http prefix, etc.)
```

Comment:
@Demmonius Is there a better (shorter) way to do this?

Can't be variables automatically added to the ENV?

```
${{ env.VERCEL_DEPLOYMENT_DOMAIN }}

instead of 

${{ fromJson(steps.restore-variable.outputs.variables).VERCEL_DEPLOYMENT_DOMAIN }}
```

It would simplify writing a lot.

Reply:
Something like https://github.com/marketplace/actions/add-env-vars might help

Reply:
Yes that's possible, I'll try to implement it asap